crates/lib/src/lib.rs

```diff
@@ -6,4 +6,5 @@ pub mod fs;
 pub mod helpers;
 pub mod manifests;
 pub mod ops;
-pub mod targets;
+
+pub use detect_targets as targets;
```

Comment:
Done!

Now it only re-export `detect_target::get_desired_targets` because `crate/bin/src/entry.rs` uses that function and I don't want to add another dependency to the binary.